src/astro/sql/operators/load_file.py

```diff
@@ -34,7 +34,7 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
-        self.output_table: Table = output_table
```

Comment:
This isn't needed because, we already have typehint for `output_table` in constructor